### DISCUSSION TITLE
Admin is not able to upload odk form by clicking on add logic button

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [samagra-comms/community#50] [General Discussion]
@abhay-samagra @ ChakshuGautam @ aayushkedia95 @SaketSinha

We are unable to add ODK form on sunbird ED staging env, please let us know when can we connect and discuss on this issue.

Jira id  - https://github.com/orgs/Sunbird-Ed/discussions/508

**Steps to reproduce:-**
1. Login with orgadmin credential on sunbird ED staging env
3. Click on profile icon
4. Click on communication console
5. Click on add new button
6. Fill the all mandatory fields then click on next button
7. Click on add logic button
8. Fill the name and upload the odk form 
9. Observe

Actual result : user is not able to add odk form by clicking on add logic button
Expected result : user should able to add odk form by clicking on add logic button

https://github.com/samagra-comms/community/assets/58033030/461b19ab-9634-4dc1-9692-0e0204b909fb

cc @chitranshu-keshav  @vinukumar-vs @rajeevsathish

**Comment by ashokreddy1208** [comment]
@abhay-samagra @ ChakshuGautam @ aayushkedia95 @SaketSinha Can you please respond to this thread
cc @chitranshu-keshav @vinukumar-vs @rajeevsathish

**Reply by abhay-samagra** [reply]
This has been fixed 3 days ago.
https://github.com/samagra-comms/uci-apis/issues/114

**Comment by abhay-samagra** [comment]
Yes, I will check this ticket and respond.Sent from my iPhoneOn 22-Jun-2023, at 10:53 AM, ashokreddy1208 ***@***.***> wrote:﻿
@abhay-samagra @ ChakshuGautam @ aayushkedia95 @SaketSinha Can you please respond to this thread
cc @chitranshu-keshav @vinukumar-vs @rajeevsathish
